doc/user/content/sql/create-connection.md

```diff
@@ -27,7 +27,8 @@ certificates) can be specified as plain `text`, or also stored as secrets.
 
 An Amazon Web Services (AWS) connection provides Materialize with access to an
 Identity and Access Management (IAM) user or role in your AWS account. You can
-use AWS connections to perform [bulk exports to Amazon s3](/serve-results/s3/).
+use AWS connections to perform [bulk exports to Amazon s3](/serve-results/s3/),
+or perform [authentication with an Amazon MSK cluster](#kafka).
```

Comment:
```suggestion
or perform [authentication with an Amazon MSK cluster](#kafka-aws-connection).
```

Added an anchor below so that we link to the specific row.  (the way our docs currently scroll past the anchors is definitely on my list of todos).